widgets/use-assets/src/runtime/widget.tsx

```diff
@@ -0,0 +1,54 @@
+/** @jsx jsx */
+import {React, AllWidgetProps,  jsx } from 'jimu-core'
+import {Icon} from 'jimu-ui'
+import {Config} from './config'
+const smallImage = require('./small-image.png')
+
+const svg2 = require('./mysvg2.svg')
+const svg3 = require('./assets/icons/mysvg3.svg')
+
+export default function(props: AllWidgetProps<Config>){
+  return (
+    <div className="widget-use-assets jimu-widget m-2" style={{overflow: 'auto'}}>
```

Comment:
```suggestion
    <div className="widget-use-assets jimu-widget" style={{overflow: 'auto'}}>
```

Removing the `m-2` bootstrap margin class so that the scroll bar appears.